Source/Core/VideoCommon/RenderState.cpp

```diff
@@ -169,38 +182,39 @@ void BlendingState::Generate(const BPMemory& bp)
 
 void BlendingState::ApproximateLogicOpWithBlending()
 {
-  // Any of these which use SRC as srcFactor or DST as dstFactor won't be correct.
-  // This is because the two are aliased to one another (see the enum).
   struct LogicOpApproximation
   {
     bool subtract;
-    BlendMode::BlendFactor srcfactor;
-    BlendMode::BlendFactor dstfactor;
+    SrcBlendFactor srcfactor;
+    DstBlendFactor dstfactor;
   };
+  // TODO: This previously had a warning about SRC and DST being aliased and not to mix them,
+  // but INVSRCCLR and INVDSTCLR were also aliased and were mixed.
+  // Thus, NOR, EQUIV, INVERT, COPY_INVERTED, and OR_INVERTED duplicate(d) other values.
```

Comment:
I split `BlendFactor` into `SrcBlendFactor` and `DstBlendFactor` since `DstClr`/`InvDstClr` and `SrcClr`/`InvSrcClr` have differences in behavior and are worth treating differently, and I found that some entries in this table use `InvDstClr`/`InvSrcClr` even though they're not actually available.  I'm not sure exactly what to do here, but the behavior here may be (and have been) wrong.